framework/audio/CMakeLists.txt

```diff
@@ -40,59 +38,55 @@ elseif(PLATFORM_OSX)
 endif()
 
 set(MODULE_SRC
-    ${SOLOUD_SRC}
     ${DRIVER_SRC}
     ${CMAKE_CURRENT_LIST_DIR}/audiomodule.cpp
     ${CMAKE_CURRENT_LIST_DIR}/audiomodule.h
     ${CMAKE_CURRENT_LIST_DIR}/audioerrors.h
-    ${CMAKE_CURRENT_LIST_DIR}/audiotypes.h
-    ${CMAKE_CURRENT_LIST_DIR}/iaudioplayer.h
     ${CMAKE_CURRENT_LIST_DIR}/iaudioengine.h
+    ${CMAKE_CURRENT_LIST_DIR}/iaudioinsert.h
+    ${CMAKE_CURRENT_LIST_DIR}/iaudiostream.h
+    ${CMAKE_CURRENT_LIST_DIR}/isequencer.h
     ${CMAKE_CURRENT_LIST_DIR}/iaudiosource.h
+    ${CMAKE_CURRENT_LIST_DIR}/iplayer.h
+    ${CMAKE_CURRENT_LIST_DIR}/iaudioplayer.h
+    ${CMAKE_CURRENT_LIST_DIR}/imidiplayer.h
+    ${CMAKE_CURRENT_LIST_DIR}/internal/src.h
```

Comment:
According to code style, we first register cpp than h

Reply:
fixed